designs/add-recap-search-option-about-representation/readme.md

```diff
@@ -0,0 +1,113 @@
+- **Add** code that examines each docket party's type to determine if the party is a petitioner.  For more details about which types will be considered to be a petitioner, see the file [`party types that will be considered to be petitioners.md`](./party%20types%20that%20will%20be%20considered%20to%20be%20petitioners.md).
+- When a petitioner party's full name value is contained within the attorney's full name value or vice versa, then the code will set the docket attribute for `petitioner represented` to true.
+- At a high level, the Django script will use the same sorts of comparisons that are represented in the following SQL to determine whether a docket's `petitioner represented` attribute should be set to true.  (This SQL will not be part of the proposed solution.  The Django script will not run this SQL.  This SQL is only to illustrate the sorts of comparisons the Django script will do.)
+
+``` sql
+UPDATE search_docket 
+SET petitioner_represented = TRUE
+WHERE EXISTS ( -- at least one "petitioner" whose name is not within its attorney's name or vice versa
+    SELECT 1
+    FROM people_db_role
+        ,people_db_party
+        ,people_db_partytype
+        ,people_db_attorney
+    WHERE search_docket.id           = people_db_role.docket_id
+    AND   people_db_role.party_id    = people_db_party.id
+    AND   people_db_role.party_id    = people_db_partytype.party_id
+    AND   people_db_role.attorney_id = people_db_attorney.id
+    AND   (people_db_party.name       NOT LIKE ('%' || people_db_attorney.name || '%')
+           OR people_db_attorney.name NOT LIKE ('%' || people_db_party.name || '%')
+    AND   people_db_partytype.name  {{{{see the link above about types that will be considered to be petitioners}}}}
+)
+```
+
+## Proposed solution: Solr
+
+**Add** a field of type boolean that corresponds to the new search_docket table column 'petitioner_represented'.  
+
+Manually add the field to Solr's configuration files (rather than adding it, for example, via Django code).
+
+The field would be represented as follows in the correct configuration file:
+
+``` xml
+<field name="petitioner_represented"
+       type="boolean"
+       indexed="true"
+       stored="true"
+       multiValued="false"/>
+```
+
+OTHER DETAILS T.B.D.
+
+## Proposed solution: Already-stored docket data
+
+**Create** a Django script that will iterate through all dockets that have already been stored and correctly set each docket's new `petitioner represented` attribute.  To determine the value, The Django script will use the new code that is described in the sub-section above about docket parsing and storage.
+
+## Proposed solution: Documentation
+
+**Revise** the documentation to reflect the above changes.
+
+DETAILS T.B.D.
+
+QUESTION:  Where is this documentation? (URL(s) are needed, please)
```

Comment:
Mostly this just [lives on the FAQ](https://www.courtlistener.com/faq/) or in (?) popups.